src/app/accounting/closing-entries/closing-entries.component.scss

```diff
@@ -1,3 +1,6 @@
 table {
   width: 100%;
+  tr.select-row:hover {
```

Comment:
```suggestion
  .select-row:hover {
```

Reply:
does not work after removing tr

Reply:
Okay, but it does in other modules. Let that be for now.